<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoostTraining</class>
 <widget class="QWidget" name="BoostTraining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>30</y>
     <width>488</width>
     <height>784</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="train_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>448</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#train_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#train_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>训练</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="times_RadioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>抓握次数</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="duration_RadioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>治疗时间</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="times_SpinBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>44</y>
       <width>131</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="duration_SpinBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>94</y>
       <width>131</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>49</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>98</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="moveType_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>246</y>
      <width>451</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#moveType_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#moveType_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>活动类型</string>
    </property>
    <widget class="QPushButton" name="fourFinger_Btn">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>41</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/boostFourNoSelected.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/boostFourHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="oneFinger_Btn">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>41</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/boostOneNoSelected.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/boostOneHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timeInterval_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>422</y>
      <width>451</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#timeInterval_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#timeInterval_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>训练难易程度</string>
    </property>
    <widget class="QPushButton" name="level3_Btn">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level3NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level3Hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level1_Btn">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level1NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level1Hover.png);}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="level2_Btn">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>40</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/level2NoSelect.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/level2Hover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="start_Btn">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>845</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_Btn">
   <property name="geometry">
    <rect>
     <x>1489</x>
     <y>845</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 8px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 8px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="LineChart" name="chart_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>760</y>
     <width>1091</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关节活动度%</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>377</width>
     <height>590</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前抓握次数</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentDuration_Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>144</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentTimes_Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>74</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>次</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="buckling_ProgressBar">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>274</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk
{
background:#70E086;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="stretch_ProgressBar">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>274</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk
{
background:#C495F5;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333333;</string>
    </property>
    <property name="text">
     <string>伸展</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333333;</string>
    </property>
    <property name="text">
     <string>屈曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前抓握时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>268</y>
      <width>36</width>
      <height>139</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/source/dividingRule/vertial.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>268</y>
      <width>36</width>
      <height>139</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/source/dividingRule/vertial.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stretch_Label">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>220</y>
      <width>74</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buckling_Label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>74</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>29</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#333333;</string>
   </property>
   <property name="text">
    <string>助力运动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CTrainingHand" name="hand_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>111</y>
     <width>581</width>
     <height>585</height>
    </rect>
   </property>
   <widget class="QLabel" name="direction_Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>364</y>
      <width>70</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/source/image/bluckling.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="timeOut_ProgressBar">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>710</y>
     <width>360</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#timeOut_ProgressBar{border:none;background:#888888; border-radius:10px;}
QProgressBar::chunk 
{
      background-color:#70E086;
      border-radius:10px;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>654</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>超时前请按压/弯曲选择的手指</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainTips_Label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>48</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>准备中...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTrainingHand</class>
   <extends>QWidget</extends>
   <header>ctraininghand.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineChart</class>
   <extends>QWidget</extends>
   <header>linechart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
